### DISCUSSION TITLE
Dose it work with item adder plugin

### DISCUSSION THREAD

**Discussion by DeathEnderMan** [Kas-tle/java2bedrock.sh#13777] [Converter Help]
Dose it work with item adder plugin and there addons like better ranks

**Comment by typicalsmc** [comment]
yes and no

**Reply by DeathEnderMan** [reply]
do you know how to do it

**Reply by DeathEnderMan** [reply]
anyone have a clue

**Reply by Sys-h-sudo** [reply]
I apologize for the bump, but you'd simply run /iazip, navigate to the output of the pack, and upload it here. This will do about 80% of the work for you, and if you're lucky it'll do it all. 

Please note you should be absolutely confident that every single config from IA is 100% up to speed and works.
